themes/jb/layouts/partials/community/card.html

```diff
@@ -9,13 +9,11 @@
 {{ else }}
   {{ $link = .Permalink }}
 {{ end }}
-<a href="{{ $link }}" {{ if ne (urls.Parse .Site.BaseURL).Host (urls.Parse $link).Host }}target="_blank"{{ end }} class="custom__is-fullheight" style="display: block; min-height: 20rem;">
-  <div class="card is-flex is-flex-direction-column is-justify-content-center is-align-items-center is-fullheight custom__is-fullheight">
-    <div class="card-image is-flex is-flex-direction-column is-justify-content-center is-align-items-center custom__is-fullwidth" style="flex: 1 0 33%;">
-      <figure class="image is-four-fifths" style="width: 65%;">
-        <img src="{{ .Params.logo }}" alt="{{ .Title | lower }} logo" class="is-centered custom__is-fullwidth py-2"
-          >
-      </figure>
+<a href="{{ $link }}"  {{ if ne (urls.Parse .Site.BaseURL).Host (urls.Parse $link).Host }}target="_blank"{{ end }}>
+  <div class="card is-fullheight" style="height: 100%;">
+    <div class="card-image is-flex is-flex-direction-column is-justify-content-center is-align-items-center" style="height: 250px;">
```

Comment:
Can you swap this div out for a `<figure>` tag?

It's supposed to be better for accessibility: https://www.pluralsight.com/blog/creative-professional/whats-difference-image-figure-tags-html5

Reply:
Actually this would fall under a decorative img and as such really should have just a blank alt="" attribute. 

https://www.w3.org/WAI/tutorials/images/

Reply:
Huh, interesting is that because *technical* they don't "add value" to the page? They're just visual indicators to quickly identify at a glance which one someone wants to click on?

I'd always thought you just give everything alt text text if you can, so people know what the image is 🤔